### DISCUSSION TITLE
HOW TO REMOVE                                    We support Ukraine and condemn war. Push Russian government to act against war. Be brave, vocal and show your support to Ukraine. Follow the latest news HERE

### DISCUSSION THREAD

**Discussion by eresers** [DirectoryLister/DirectoryLister#894] [Help & Support] [answered]
How to remove it?
![image](https://user-images.githubusercontent.com/106179270/170097843-5c889d14-f312-48ef-b811-d7a05d368ecd.png)

**Comment by PHLAK** [comment, marked as answer]
This isn't coming from Directory Lister.

**Comment by ezirs** [comment]
<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>

function onLoad() {
        $('body').find('div[class$="disclaimer"]').remove();
    }
